.github/workflows/tests.yml

```
@@ -21,129 +21,77 @@ jobs:
         product: [CHROMIUM, FIREFOX, WEBKIT]
         os: [windows-latest, ubuntu-latest, macos-latest]
     steps:
-      - name: Install Media Pack
-        if: ${{ matrix.os == 'windows-latest' }}
-        shell: powershell
-        run: Install-WindowsFeature Server-Media-Foundation
       - uses: actions/checkout@v2
-      - uses: microsoft/playwright-github-action@v1
-      - name: Setup .NET Core (On Mac)
-        if: ${{ matrix.os == 'macos-latest' }}
-        uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.100
-        env:
-          DOTNET_INSTALL_DIR: /users/runner/alternate/dotnet
-      - name: Setup .NET Core (Not on mac)
-        if: ${{ matrix.os != 'macos-latest' }}
+          submodules: true
+          fetch-depth: 0
+      - name: Setup .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.100
-      - name: Create Certificate
+          dotnet-version: 5.0.x
+      - name: Install prerequisites and download drivers
+        shell: pwsh
+        run: .\build.ps1 driver
+      - name: Building...
+        shell: pwsh
+        run: dotnet build 
+        working-directory: src
+      - name: Installing Browsers and dependencies...
+        shell: pwsh
         run: |
-          dotnet dev-certs https --clean
-          dotnet dev-certs https -ep src/Playwright.Tests.TestServer/testCert.cer
-      - name: Clean
-        run: dotnet clean ./src/Playwright.sln --configuration Debug && dotnet nuget locals all --clear
-      - name: Download Drivers
-        run: dotnet run -p ./src/tools/Playwright.Tooling/Playwright.Tooling.csproj -- download-drivers --basepath .
-      - name: Download Browsers
-        run: dotnet run -p ./src/Playwright -f net5.0 install
-      - name: Run tests (!Linux)
+          dotnet run -p .\src\Playwright\Playwright.csproj -f net5.0 -- install
```

```suggestion
          playwright install
```
why does this not work yet?

---

The CLI isn't installed. Nor do I want to install it - to avoid running into weird issues with versioning/NuGet packages.